Documentation/SIL.ReleaseTasks.md

```diff
@@ -125,6 +119,8 @@ conventions.
 
 - `Value` (output parameter): The name of the property that will be set
 
+- 'FilterEntries': Filters out entries that do not belong to a specific nuget package, when enabled.
```

Comment:
That should be backticks instead of single quotes:

```suggestion
- `FilterEntries`: Filters out entries that do not belong to a specific nuget package, when enabled.
```